Formula/gsoap.rb

```diff
@@ -15,12 +15,6 @@ class Gsoap < Formula
   def install
     ENV.deparallelize
```

Comment:
It is still needed. I contacted upstream via email, it's the only contact method I've found on their website.